source/util/ilist_node.h

```diff
@@ -76,11 +76,14 @@ class IntrusiveNodeBase {
   inline void RemoveFromList();
 
  protected:
-  // This function will replace |this| with |target|.  No nodes that are not
+  // Replaces |this| with |target|.  No nodes that are not
```

Comment:
Replaces |this| with |target|. |this| and |target| need to be either
- both sentinels, in which case the entire list is moved
*or*
- both not sentinels, in which case only single node is moved
No storage is deleted.

Reply:
done